focus-ios/.swiftlint.yml

```diff
@@ -0,0 +1,126 @@
+only_rules: # Only enforce these rules, ignore all others
```

Comment:
Sorry to jump in...Not sure if you have discussed this already, but ideally we will have just one .swiftlint file in the root dir for both projects... we may need this as a first step to enable rule by rule and then use the one swiftlint in root.. just trying to understand to be on the same page. Thanks!

Reply:
Yes. What's happening here is that we're basically trying to figure out how to split up tasks to get Focus in the same swiftlint place as Firefox. In order to do that, we were thinking of, temporarily, having Focus have a separate swiftlint file. That way, we can split up tasks for contributors, and then we can enabled separately. And once the rules are the same, then we can remove thiis setup and have the root folder swiftlint file be the source of truth.

IIf you can think of another way to do this easily, I'm all ears, but this iis just what we came up with.